src/screens/Overview/index.js

```diff
@@ -22,7 +23,14 @@ export default ({instructor}) => (
     aside={
       <div>
         <div className='mb4'>
-          <Stats instructor={instructor} />
+          <Request url={instructor.revenue_url}>
+            {({data}) => (
+              <Stats 
+                instructor={instructor}
+                revenue={data.revenue}
```

Comment:
Oh gotcha, I misunderstood - thought you meant the `instructor.revenue` prop. I'll update this. Thanks.